Dockerfile.compute-node

```diff
@@ -322,8 +322,9 @@ ARG PG_VERSION
 RUN curl -sSO https://static.rust-lang.org/rustup/dist/$(uname -m)-unknown-linux-gnu/rustup-init && \
     chmod +x rustup-init && \
     ./rustup-init -y --no-modify-path --profile minimal --default-toolchain stable && \
-    rm rustup-init && \
-    cargo install --git https://github.com/vadim2404/pgx --branch neon_abi_v0.6.1 --locked cargo-pgx && \
+    rm rustup-init
+
+RUN cargo install --locked --version 0.7.3 cargo-pgx && \
```

Comment:
I'm not sure that we need to split these items into two different layers. WDYT?

Reply:
Sure, merged them back